spec/statsd_spec.rb

```diff
@@ -13,8 +13,6 @@ class Datadog::Statsd
     @statsd.socket = FakeUDPSocket.new
   end
 
-  after { @statsd.socket.clear }
```

Comment:
I'm pretty sure this isn't necessary because the `before` clause always initializes a new socket.  If I can remove it, I can also remove the `after` clause in the `#close` tests, without having to mock out `#clear`.